sample/QueueTrigger-Powershell/queueTrigger.ps1

```diff
@@ -1,8 +1,4 @@
-$in = Get-Content $Env:input
-
-[Console]::WriteLine("Powershell script processed queue message '$in'")
-
-$output = $Env:output
-$json = $in | ConvertFrom-Json
-$entity = [string]::Format('{{ "Status": 0, "Title": "Powershell Table Entity for message {0}" }}', $json.id)
+$in = Get-Content $inputdata

```

Comment:
If you used $Env:input here, can we avoid the breaking change of renaming input to inputdata, and retain consistency across the languages?